v2/controllers/marketplace/meterbase_controller.go

```diff
@@ -1121,12 +1337,24 @@ func (r *MeterBaseReconciler) uninstallMetricState(
 ) []ClientAction {
 	deployment, _ := factory.MetricStateDeployment()
 	service, _ := factory.MetricStateService()
-	sm, _ := factory.MetricStateServiceMonitor()
+	sm0, _ := factory.MetricStateServiceMonitor()
```

Comment:
sm0 is fine here for the delete, small enough function to know what we're doing.

Reply:
ok